securedrop_client/gui/widgets.py

```diff
@@ -1576,26 +1582,13 @@ def on_star_update_successful(self, source_uuid: str) -> None:
             self.pending_count = self.pending_count - 1
 
 
-class LoginOfflineLink(QLabel):
-    """
-    A button that logs the user in in offline mode.
-    """
-
-    clicked = pyqtSignal()
+class LoginOfflineLink(SDPushButton):
+    """A button that logs the user in, in offline mode."""
 
     def __init__(self) -> None:
-        # Add svg images to button
         super().__init__()
-
-        # Set css id
-        self.setObjectName("LoginOfflineLink")
-
-        self.setFixedSize(QSize(120, 22))
-
         self.setText(_("USE OFFLINE"))
-
-    def mouseReleaseEvent(self, event: QMouseEvent) -> None:
-        self.clicked.emit()
+        self.setAlignment(SDPushButton.AlignLeft)
 
 
 class SignInButton(QPushButton):
```

Comment:
Why not inherit from `SDPushButton` here too? Are your plans for a follow-up PR?

Reply:
Yes, you read my mind :wink:
Supporting two styles of buttons seemed like more code to review at once, and can be done in separate PRs.